sqlalchemy_utils-stubs/primitives/country.pyi

```diff
@@ -1,5 +1,19 @@
+from __future__ import annotations
+
+from typing import Union
+
 from .. import i18n as i18n
 from ..utils import str_coercible as str_coercible
-from typing import Any
 
-def __getattr__(name: str) -> Any: ...  # incomplete
+
+@str_coercible
+class Country:
+    def __init__(self, code_or_country: Union[str, Country]) -> None:
+        pass
+
+    def name(self) -> str:
+        pass
+
+    @classmethod
+    def validate(self, code: str) -> None:
+        pass
```

Comment:
```suggestion
        pass

```

Please make sure all files [end with a newline](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) (so same thing for the other files)